src/main/shell/cac-git

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [[ -n "${JAVA_HOME}" ]] && [[ -x "${JAVA_HOME}/bin/java" ]];  then
+    JAVA="${JAVA_HOME}/bin/java"
+elif type -p java; then
+    JAVA=java
+else
+    echo "Error: JAVA_HOME is not set and java could not be found in PATH." 1>&2
+    exit 1
+fi
+
+# http://stackoverflow.com/questions/59895/can-a-bash-script-tell-what-directory-its-stored-in
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  TARGET="$(readlink "$SOURCE")"
+  if [[ $SOURCE == /* ]]; then
+    SOURCE="$TARGET"
+  else
+    PROJECT_BIN_DIR="$( dirname "$SOURCE" )"
+    SOURCE="$PROJECT_BIN_DIR/$TARGET" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+  fi
+done
+RDIR="$( dirname "$SOURCE" )"
+PROJECT_BIN_DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+PROJECT_DIR="$(dirname "$PROJECT_BIN_DIR")"
+
+${JAVA} -jar /usr/local/lib/cac-git/cac-aware-jgit-jar-with-dependencies.jar ${1+"$@"}
```

Comment:
Jon, looks like the script works to compute the PROJECT_DIR, but then uses a hard coded path to fidn the .jar? Was that variable supposed to be used to find the .jar?

Reply:
The first iteration of the script installed into `/opt/cac-git` with `bin` and `lib` subdirectories.  The boilerplate was to resolve the library.  However, I changed my mind and decided to install into a more standard location so it would already be on the path.  I forgot to remove the boilerplate.  Thinking about it more, we should probably either put it into `/usr/lib` if the binaries are eventually published rather than `/usr/local/lib` since the latter seems to be used more for when building and installing manually from source.

From a conversation with @ryankenney, a `.deb` package probably makes more sense since a good portion of users are on ubuntu.